tests/test_parsers.py

```diff
@@ -470,8 +470,11 @@ def test_strict_parsing(self):
             parse_sssom_table(stream, strict=True)
 
         # Make sure it parses in non-strict mode
-        msdf = parse_sssom_table(stream)
-        self.assertEqual(len(msdf.df), 2)
+        with open(input_path, "r") as file:
```

Comment:
Why open the file again? it seemed to have worked before?

Reply:
Now, we only consume the file once and don't put it into a buffer that we can seek to the top of. The old way of reading the same stream twice was also problematic since you shouldn't assume this is even possible

Reply:
ok